### ISSUE TITLE
No more support for ARM32 (Docker Images)?

### ISSUE THREAD

**Issue by ssj4maiko** [zedeus/nitter#828]
I looked into some issues here, and found some conversation.

I'm using an RP4, and although I'm aware that it should be ARM64 complaint, IIRC, at the time I installed it, I think Raspbian OS 64 was still experimental, so I'm pretty sure I'm using a 32 bit one _(I had problems with NextCloud previously, who accidentally dropped support, fixed it, but they announced they are dropping support to ARM32)_.

I'm currently using unixfox's docker image, I'm using his latest version, I was having some problems but I found the solution in https://github.com/zedeus/nitter/issues/778.

But I noticed it has been over a month since his latest update, and in that same Issue, at the end, he mentions to use the official docker image, so I'm guessing he may be dropping off his quay.io updates?

I found a bug, which I found having been seemingly reported here already (https://github.com/zedeus/nitter/issues/768), although I guess my difference is that I'm getting a similar error _(no garbage characters though)_ on a single twitter profile, everything else works fine, so I though about updating to confirm its status and then write an issue in case I confirmed it not working.

However if I can't update because of a lack of support, I guess I will have to accept that I won't be able to fix my nitter problems from here on and use normal twitter on those instances.

**Comment by LainLayer** [comment]
I am currently running a nitter instance on my beaglebone with armv7l
I built it from source and the only issue i encountered is banner images are not showing up. Never tried the docker image in question

**Comment by SethPyle376** [comment]
64 bit Raspbian is out of beta, I'd recommend switching your Pi4  to that for a variety of reasons

https://www.raspberrypi.com/news/raspberry-pi-os-64-bit/

**Comment by ssj4maiko** [comment]
OK, finally took some time and did the deed. Took some time, but it worked.
Tried nitter again I was having some problems, but I fixed it by setting `:latest-arm64`

My docker-compose also didn't work (it couldn't connect to redis for whatever reason), but the `docker-compose.yml` supplied by the project (https://github.com/zedeus/nitter/blob/master/docker-compose.yml) worked well (I just adapted the variables to what I had before), and now it's working fine.
